src/lists/mainnet/index.ts

```diff
@@ -0,0 +1,13 @@
+import {WRAPPED_21CO_TOKENS} from './21co-tokens';
+import {INDEX_COOP_MAINNET_TOKENS} from './index-tokens';
+import {MainnetTokens} from './mainnet';
```

Comment:
Using this structure we could easily exchange the mainnet tokens from a different source - if we want to in the future. @0xonramp